satellite/satellitedb/orders.go

```diff
@@ -39,6 +43,16 @@ func (db *ordersDB) UseSerialNumber(ctx context.Context, serialNumber storj.Seri
 	)
 	_, err := db.db.ExecContext(ctx, statement, storageNodeID.Bytes(), serialNumber.Bytes())
 	if err != nil {
+		// TODO maybe move to dbutil
```

Comment:
That sounds like a good idea, maybe into separate though, e.g. `pgutil.IsContraintError(err) || sqliteutil.IsConstraintError(err)`... othewise storagenode might accidentally start importing postgres

Reply:
done